### ISSUE TITLE
Doesn't work with Next.js styled jsx

### ISSUE THREAD

**Issue by nblthree** [windicss/windicss-webpack-plugin#15]
```
error - ./components/Layout.jsx:13:8
Syntax error: Unexpected token, expected "}"

  11 | `
  12 |         body {
> 13 |   margin: 0;
     |         ^
  14 |   padding: 0;
  15 |   font-family: '-apple-system', 'BlinkMacSystemFont', 'Segoe UI',
  16 |               'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
```

**Comment by harlan-zw** [comment]
Hey @nblthree 

Thanks for the issue. I don't work with Next.js or jsx, would you be able to help me look into this?

What does the Layout.jsx file look like?

**Comment by nblthree** [comment]
simple stuff

```
import Head from 'next/head'

export default function Layout({ title, children }) {
  return (
    <div id="layout-wrapper">
      <Head>
        <title>{title}</title>
      </Head>
      {children}
      <style jsx global>{`
        * {
          box-sizing: border-box;
        }
        body {
          margin: 0;
          padding: 0;
          font-family: '-apple-system', 'BlinkMacSystemFont', 'Segoe UI',
            'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
            'Helvetica Neue', 'sans-serif';
          -webkit-font-smoothing: antialiased;
          -moz-osx-font-smoothing: grayscale;
          width: 100vw;
          height: 100vh;
          overflow: hidden;
        }
        // And so on .........
      `}</style>
    </div>
  )
}
```

**Comment by nblthree** [comment]
I don't know if the loader is applied to the above but maybe styled-jsx should be ignored too here https://github.com/windicss/windicss-webpack-plugin/blob/master/src/loaders/transform-template.ts#L37

**Comment by nblthree** [comment]
I think that it should be handled with Vue SFC's `<style>`

**Comment by harlan-zw** [comment]
This one should now be resolved in v0.4.0, thanks again for your help @nblthree
